Admin/ORM/MediaAdmin.php

```diff
@@ -36,7 +36,7 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('context', null, array(
                 'show_filter' => $this->getPersistentParameter('hide_context') !== true,
             ), 'choice', $options)
-            ->add('category', null, array(
+            ->add('category', 'doctrine_orm_choice', array(
```

Comment:
I think your best chance at solving the bug is starting [here](https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/Guesser/TypeGuesser.php#L26). The `null` should be converted in this method IMO.

> I don't know how it did worked before, now it does not :)

If you look at the history of the file, `null` was always here, implicitly or by default.

> Or you can do it, I don't mind -> issue exists, I think that you can reproduce it.

Sorry but I don't even use these bundles so that would be way too cumbersome for me, leaving aside the fact that I have zero benefit in solving this bug.